docker/scout/scripts/main_ftm.py

```diff
@@ -44,13 +61,25 @@ def update_wallets_gauge(
         eth_balance = float(WEB3_INSTANCE.fromWei(
             WEB3_INSTANCE.eth.getBalance(wallet_address), "ether")
         )
-
+        token_address = token_address.lower()
+        weth_address = COINGECKO_TOKENS['WETH'].lower()
+        # Amount of tokens in wallet
         wallets_gauge.labels(
             wallet_name, wallet_address, token_name, token_address, "balance"
         ).set(token_balance)
+        # Amount of ETH in wallet
```

Comment:
So this was about special handling for the gas token which isn't an ERC-20.  In the case of Etherium it's ETH.  In the case of Fantom it's FTM. 

Weth should be well handled by the normal logic in this case.   

There's also a WFTM that is easier to work with as an ERC-20 version of FTM that is always wrappable/unwrappable.

So on Etherium/Arbitrum ETH/WETH  is like FTM/WTFM on Fantom, or MATIC/wMATC on polygon.